lib/src/generate/build_impl.dart

```diff
@@ -294,3 +305,155 @@ class BuildImpl {
     }
   }
 }
+
+/// Extends [BuildImpl] to add the [runWatch] method, which watches all
```

Comment:
Ya I only used inheritance in order to get at the private variables, because I forgot momentarily how privacy works in Dart lol.